tybalt/models.py

```diff
@@ -254,8 +276,8 @@ class Adage(BaseModel):
     Usage: from tybalt.models import Adage
     """
     def __init__(self, original_dim, latent_dim, noise=0.05, batch_size=50,
-                 epochs=100, sparsity=0, learning_rate=1.1,
-                 loss='mse'):
+                 epochs=100, sparsity=0, learning_rate=0.0005,
```

Comment:
Why the change in learning rate (just curious)?

Reply:
I was initially using `Adadelta` as the optimizer to train ADAGE. This worked well after a parameter sweep, but i found that it tended to work poorly with default parameters. `Adam`, which is the optimizer I use for Tybalt, is much more well behaved with different data input under default parameters. For the simulation, I was finding ADAGE to have uniformly poor performance, which was concerning. After some troubleshooting I discovered what I described above and results have improved. I want to compare algorithms, not optimizers here so I chose to keep them consistent - although this is definitely a limitation of comparing non-convex optimization driven algorithms to things like PCA!